interfaces/repo_test.go

```diff
@@ -910,3 +939,28 @@ func (s *RepositorySuite) TestSecuritySnippetsForSnapFailure(c *C) {
 	c.Assert(err, ErrorMatches, "cannot compute snippet for consumer")
 	c.Check(snippets, IsNil)
 }
+
+func (s *RepositorySuite) TestSecuritySnippetsForSnapFailureWithStaticSnippets(c *C) {
+	var testSecurity SecuritySystem = "security"
+	iface := &TestInterface{
+		InterfaceName: "interface",
+		StaticSlotSecuritySnippetCallback: func(slot *Slot, securitySystem SecuritySystem) ([]byte, error) {
+			return nil, fmt.Errorf("cannot compute static snippet for consumer")
+		},
+		StaticPlugSecuritySnippetCallback: func(plug *Plug, securitySystem SecuritySystem) ([]byte, error) {
+			return nil, fmt.Errorf("cannot compute static snippet for provider")
```

Comment:
I've added a TODO to do a quick pass over all the test data to use consistent wording.

I wonder what that wording should be though? Any suggestions